pkg/storaged/nfs-details.jsx

```diff
@@ -80,10 +80,10 @@ function get_exported_directories(server) {
 
 export function nfs_fstab_dialog(client, entry) {
     const mount_options = entry ? entry.fields[3] : "defaults";
-    const split_options = format.parse_options(mount_options == "defaults" ? "" : mount_options);
-    const opt_auto = !format.extract_option(split_options, "noauto");
-    const opt_ro = format.extract_option(split_options, "ro");
-    const extra_options = format.unparse_options(split_options);
+    const split_options = parse_options(mount_options == "defaults" ? "" : mount_options);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230926-135906-360ec0bd-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)